operations/iuf/workflows/deploy_product.md

```diff
@@ -10,13 +10,31 @@
 section of the _HPE Cray EX System Software Stack Installation and Upgrade Guide for CSM (S-8052)_ provides a table that summarizes which product documents contain information or actions for the `deploy-product` stage.
 Refer to that table and any corresponding product documents before continuing to the next step.
 
-1. Invoke `iuf run` with `-r` to execute the [`deploy-product`](../stages/deploy_product.md) stage.
+1. Invoke `iuf run` with activity identifier `${ACTIVITY_NAME}` and use `-r` to execute the [`deploy-product`](../stages/deploy_product.md) stage. Perform the upgrade using product content found in `${MEDIA_DIR}`.
 
-    (`ncn-m001#`) Execute the `deploy-product` stage.
+   - Option 1: Install or upgrade without SLURM or PBS Products
 
-    ```bash
-    iuf -a "${ACTIVITY_NAME}" run -r deploy-product
-    ```
+      (`ncn-m001#`) Execute the `deploy-product` stage.
+
+      ```bash
+      iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run -r deploy-product
+      ```
+
+   - Option 2: Enable SLURM or PBS Pro Products when installing USS 1.1 or higher
+
+     **`NOTE`** This subsection is mandatory only while installing a workload manager with USS 1.1 or higher. Additional arguments are available to control the behavior
+     of the `deploy-product` stage (for example, `-rv`). See the [`deploy-product` stage documentation](../stages/deploy_product.md) for details and adjust the example
+     below if necessary.
+
+     1. Select which workload manager to use on the system (either SLURM or PBS). For more information, see the `deploy-product` stage details in the "Install and
+    Upgrade Framework" section of the _HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)_.
+
+     1. (`ncn-m001#`) Execute the `deploy-product` stage. Use site variables from the `site_vars.yaml` file found in `${ADMIN_DIR}` and recipe variables from the `product_vars.yaml` file found in `${ADMIN_DIR}`.
+
+        ```bash
+        iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run --site-vars \
+        "${ADMIN_DIR}/site_vars.yaml" -bpcd "${ADMIN_DIR}" -r deploy-product
+        ```
```

Comment:
```suggestion

     (`ncn-m001#`) Execute the `deploy-product` stage. Use site variables from the `site_vars.yaml` file found in `${ADMIN_DIR}` and recipe variables from the `product_vars.yaml` file found in `${ADMIN_DIR}`.

     ```bash
     iuf -a ${ACTIVITY_NAME} -m "${MEDIA_DIR}" run --site-vars \
     "${ADMIN_DIR}/site_vars.yaml" -bpcd "${ADMIN_DIR}" -r deploy-product
     ```
```

Lines 29-31 only apply to the deliver-product stage and can be removed here. There is no need to select the workload manager again. For this reason, I removed the step number and extra indentation for lines 32-37 as well.